traitsui/examples/demo/Standard_Editors/DataFrameEditor_demo.py

```diff
@@ -11,6 +11,8 @@
   missing.
 
 -------------------------------------------------------------------------------
+
+To run this demonstration successfully, you must have **pandas** installed.
```

Comment:
I almost did not add this sentence, but then DataFrameEditor is not necessarily about "pandas", so many things can be called DataFrame.  This example is missing a better description but that's orthogonal.